cliport/tasks/packing_google_objects.py

```diff
@@ -5,13 +5,13 @@
 import numpy as np
 from cliport.tasks.task import Task
 from cliport.utils import utils
-from cliport.utils import load_chicago_data
+#from cliport.utils import load_chicago_data
 from cliport.utils import command_strings
 import tempfile
 
 import pybullet as p
-import pybullet_data
-import cv2
+#import pybullet_data
```

Comment:
Could you explain why these are commented? It seems like it will break the code.